external/dockerfile_functions.sh

```diff
@@ -415,15 +415,12 @@ print_test_results() {
 print_test_script() {
     local file=$1
     local test=$2
-    local script=$3
 
     if [[ ${check_external_custom_test} -eq 1 ]]; then 
         echo -e "# This is the main script to run ${test} tests" \
-                "\nCOPY external_custom/dockerfile/${script} /${script}" \
```

Comment:
This shouldn't be removed. It should be updated as `"\nCOPY external_custom/dockerfile/test.sh /test.sh" \`. Same as line 426. Test.sh still need to be copied to docker image.